client/app/scripts/charts/node.js

```diff
@@ -83,6 +83,9 @@ export default class Node extends React.Component {
     if (this.props.pseudo) {
       classNames.push('pseudo');
     }
+    if (this.props.shape === 'cloud') {
+      classNames.push('cloud');
+    }
```

Comment:
Can we just pop the cloud class down in the cloud shape or needed for selectors up here?